sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -65,7 +67,7 @@
 
     @BeforeClass
     public static void setupClass() {
-        useIPv6 = randomBoolean();
+        useIPv6 = randomBoolean() && !"true".equalsIgnoreCase(System.getenv(NO_IPV6));
```

Comment:
would put the property check first.

Reply:
why?

Reply:
to avoid calling randomBoolean() unnecessarily

Reply:
This is called once per job and a couple of ns at most? Might be that equalsIgnoreCase and `System.getenv` call is more expensive